virttest/guest_agent.py

```diff
@@ -552,6 +552,50 @@ def set_vcpus(self, action):
         return self.cmd(cmd=cmd, args=action)
 
     @error_context.context_aware
+    def get_time(self):
+        """
+        Get the time of guest, return the time from Epoch of 1970-01-01 in UTC
+        in nanoseconds
+        """
+        cmd = "guest-get-time"
+        if not self._has_command(cmd):
+            return False
+        return self.cmd(cmd=cmd)
+
+    @error_context.context_aware
+    def set_time(self, nanoseconds=None):
+        """
+        set the time of guest, the params passed in is in nanoseconds
+        """
+        cmd = "guest-set-time"
+        args = None
+        if not self._has_command(cmd):
+            return False
+        if nanoseconds:
+            args = {"time": nanoseconds}
+        return self.cmd(cmd=cmd, args=args)
+
+    @error_context.context_aware
+    def fstrim(self):
+        """
+        Issue the cmd "guest-fstrim" to the guest agent and get the result
+        """
+        cmd = "guest-fstrim"
+        if not self._has_command(cmd):
+            return False
+        return self.cmd(cmd)
+
+    @error_context.context_aware
+    def get_network_interface(self):
+        """
+        Issue the cmd "guest-network-get-interfaces" to the guest agen
```

Comment:
I like doc to be more specific ,just like "Get list of guest IP addresses, MAC addresses ......",
You can refer the help info of the command.